server/models/campaigns.py

```diff
@@ -10,8 +10,10 @@ class CampaignModel(db.Model):
     id = db.Column(db.BigInteger, primary_key=True)
     # Figure out how to make a unique clause where the combination between user_id and campaign title must be unique
     title = db.Column(db.String(200), unique=True, nullable=False)
-    # steps = db.relationship("StepModel", backref="campaign", lazy=True)
+    steps = db.relationship("StepModel", backref="campaign", lazy=True)
     user_id = db.Column(db.BigInteger, db.ForeignKey("users.id"), nullable=False)
+    reply_count = db.Column(db.BigInteger, nullable=False, default=0)
```

Comment:
I think what we need for reply_count and prospect_count should be calculated based on what's in the prospects_step_join table at any given moment.. essentially a group_by and count per this doc.. https://stackoverflow.com/questions/1052148/group-by-count-function-in-sqlalchemy .. I'm not sure exactly how sqlAlchemy does this, but I recall this from learning SQL.